database/insertData/03_template_A_featureShowcase.js

```diff
@@ -1102,6 +1102,161 @@ exports.queries = [
+                        description: "Type the two-character country code (must be CAPS sorry)"
+                        placeholder: "Search countries"
+                        icon: "world"
+                        minCharacters: 2
+                        multiSelect: true
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query countries($code: ID!) {country(code: $code) {name, capital, emoji, code}}",
+                            "https://countries.trevorblades.com",
+                            [
+                              "code"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "search.text"
+                              ]
+                            },
+                            "country"
+                          ]
+                        }
+                        displayFormat: {
+                          title: "\${emoji} \${name}"
+                          description: "Capital: \${capital}"
+                        }
+                      }
+                    }
+                    {
+                      code: "searchATC"
+                      index: 9
+                      title: "Search ATC Codes"
+                      elementTypePluginCode: "search"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: {
+                        label: "Lookup ATC codes"
+                        description: "Start typing a drug name"
+                        placeholder: "Search medicines"
+                        icon: "pills"
+                        minCharacters: 3
+                        multiSelect: true
+                        source: {
+                          operator: "graphQL",
+                          children: [
+                            "query GetEntitiesByName($search: String!) {entities (filter: { code: \\"\\" description: $search type: \\"[drug]\\" match: \\"contains\\"},offset: 0,first: 25) {data {        code,     description,        type,        uid  }, totalLength }}",
+                            "https://codes.msupply.foundation:2048/graphql",
+                            [
+                              "search"
+                            ],
+                            {
+                              operator: "objectProperties",
+                              children: [
+                                "search.text"
+                              ]
+                            },
+                            "entities.data"
+                          ]
+                        }
+                        displayFormat: {
+                          title: "\${description}",
+                          description: "ATC code: \${code}"
```

Comment:
Just to be clearer about this one, what is returned from UC isn't the ATC code, that used to be one of the parameters in the properties array of a drug, but has been removed temporary from UC server, so now what this is showing is actually the mSupply universal code for this drug.

Reply:
Ah okay, got it. dcf2e3f